src/main/java/heronarts/lx/color/GradientUtils.java

```diff
@@ -193,25 +197,49 @@ public enum BlendMode {
     }),
 
     HSV((c1, c2, lerp) -> {
+      float hue1 = c1.hue;
+      float hue2 = c2.hue;
+      float sat1 = c1.saturation;
+      float sat2 = c2.saturation;
+      float bright1 = c1.brightness;
```

Comment:
Don't pull bright1/bright2 out here, they're never modified, just adds clutter...

Reply:
Thanks, missed that on the first cleanup commit.